.github/ISSUE_TEMPLATE/builder.md

```diff
@@ -32,7 +32,7 @@ Please consider creating an isolated reproduction repo to make it easy for the v
 
 | package                            | version |
 | ---------------------------------- | ------- |
-| `@typescript-eslint       `        | `X.Y.Z` |
+| `@angular-eslint/builder`          | `X.Y.Z` |
 | `TypeScript`                       | `X.Y.Z` |
```

Comment:
`Typescript` is already informed when you run `ng version` (something that is done just below), so I don't know if it's necessary to be redundant here.